glusterd2/main.go

```diff
@@ -119,6 +121,29 @@ func main() {
 		log.WithError(err).Fatal("Failed to generate local auth token")
 	}
 
+	// Get the Jaeger endpoints from config file
+	jaegerEndpoint := config.GetString("jaegerendpoint")
+	jaegerAgentEndpoint := config.GetString("jaegeragentendpoint")
```

Comment:
With just this, the endpoints can only be fetched from the config (toml) file. We prefer that all configurable options be configurable as CLI options as well. Please refer to how the store package initalizes its flags and config (refer `store.InitFlags`).

Reply:
I have introduced tracing.InitFlags() to initialize the jaeger endpoints using CLI as suggested.